custom/opm/utils.py

```diff
@@ -109,3 +112,12 @@ def numeric_fn(val):
     except TypeError:
         sort_val = -1
     return {'sort_key': sort_val, 'html': val}
+
+
+def format_bool(bool_or_none):
+    if bool_or_none is None:
+        return EMPTY_FIELD
+    elif bool_or_none:
+        return 'Yes'
```

Comment:
[05/08/15 2:04:20 pm] Sravan Reddy: hey anush, should ‘Yes’, or ‘No’ be in hindi?
[05/08/15 2:04:47 pm] Anushree: no this for their own analysis
[05/08/15 2:04:52 pm] Anushree: so it's all english

If I add translation, it might get transalations and display in Hindi. So keeping it as is